src/Compilers/VisualBasic/Portable/CommandLine/VisualBasicCommandLineParser.vb

```diff
@@ -1625,7 +1625,7 @@ lVbRuntimePlus:
                 Return Nothing
             End If
 
-            If fullPath Is Nothing OrElse Not PathUtilities.IsValidFilePath(fileName) Then
```

Comment:
>fullPath Is Nothing OrElse  [](start = 15, length = 27)

No need to check for null?

Reply:
`IsValidFilePath` does that check.

---
In reply to: [192185024](https://github.com/dotnet/roslyn/pull/27279#discussion_r192185024) [](ancestors = 192185024)